CHANGES.md

```diff
@@ -124,7 +124,8 @@ Release Notes.
 * Add complexity limitation for GraphQL query to avoid malicious query.
 * Add `Column.shardingKeyIdx` for column definition for BanyanDB.
 * Fix the configuration of `Aggregation` and `GC Count` metrics for oap self observability
-
+* Refactor IoTDB storage plugin, add IoTDBDataConverter and fix ModifyCollectionInEnhancedForLoop bug.
```

Comment:
Please move this to the end of backend change. This is separating  one change log.